README.md

```diff
@@ -97,5 +121,10 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
 ```
 
+'Bob'
+'Curly'
+'Mo'
+2
+
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.